glancesync/glancesync_serversfacade.py

```diff
@@ -105,7 +148,29 @@ def upload_image(self, regionobj, image):
         :param image: GlanceSyncImage object; the image to be uploaded.
         :return: The UUID of the new image.
         """
-        return _upload_image(regionobj, image, self.images_dir)
+        client = self._get_glanceclient(regionobj.region)
+        try:
+            with open(self.images_dir + '/' + image.id, 'r') as file_obj:
+                try:
+                    new_image = client.images.create(
+                        container_format=image.raw['container_format'],
+                        disk_format=image.raw['disk_format'],
+                        name=image.name, is_public=image.is_public,
+                        protected=image.raw['protected'],
+                        min_ram=image.raw['min_ram'],
+                        min_disk=image.raw['min_disk'],
+                        properties=image.user_properties, data=file_obj)
+                    return new_image.id
+                except Exception, e:
+                    msg = regionobj.fullname + ': Upload of ' + image.name +\
```

Comment:
Really?
 grep "Upload of" *.py
glancesync_serversfacade.py:                    msg = regionobj.fullname + ': Upload of ' + image.name +\

The message is not duplicated in the file nor in other modules. Perhaps you have seen in the test the other time.